### ISSUE TITLE
Lunr JS Search Internationalization

### ISSUE THREAD

**Issue by monarci** [squidfunk/mkdocs-material#345]
## Description

Am building a mkdocs site with this theme, but need to be able to search through Japanese text. I know this is an issue with lunr.js and there is a plug-in ([here](https://github.com/MihaiValentin/lunr-languages)) to work around the issue and would love to see it integrated with this theme by default. Apparently all we need to do is load the plug-in after lunr itself and then set the language when creating the search index. If you could point me in the direction as to how to build your source to the compiled versions found in the `mkdocs-material` package I'll get right to work.

### Expected behavior

Search through Japanese/non-latin text.

### Actual behavior

Only latin characters hit on search, ignoring everything else.

### Steps to reproduce the bug

1. Create a page with non-latin characters
2. Search for said text

### Package versions

* Python: 2.7.5
* MkDocs: 0.16.3
* Material: 1.6.4

### Project configuration

``` yaml
site_name: test
theme: material
copyright: Test
site_author: Test
use_directory_urls: false
markdown_extensions:
  - admonition
  - codehilite(guess_lang=false)
  - toc(permalink=true)
pages:
  - Home: index.md
  - About: about.md
```

### System information

* OS: CentOS 7

**Comment by bborysenko** [comment]
This is how I was able to add support for [indexing multi-language content](https://github.com/MihaiValentin/lunr-languages#indexing-multi-language-content) (Еnglish and Russian) - https://github.com/bborysenko/mkdocs-material/commit/6cf8eb3c55944672fd397efe2a66a8ed2817d6bf.

![screenshot 2017-05-25 17 39 57](https://cloud.githubusercontent.com/assets/3684918/26455023/896b4a20-4171-11e7-8651-40ac3fee6773.png)
![screenshot 2017-05-25 17 40 12](https://cloud.githubusercontent.com/assets/3684918/26455030/8cfe94c6-4171-11e7-9462-5bf5ca264dba.png)

What is wrong is:

**1. `.githooks/pre-commit/check.sh` fails with following message:**

```
[17:26:41] Finished 'assets:javascripts:annotate' after 899 ms
tmp/assets/javascripts/components/Material/Search/Result.jsx:27
 27: import * as lunrStemmer from 'lunr-languages/lunr.stemmer.support';
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ lunr-languages/lunr.stemmer.support. Required module not found

tmp/assets/javascripts/components/Material/Search/Result.jsx:28
 28: import * as lunrMulti from 'lunr-languages/lunr.multi';
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^ lunr-languages/lunr.multi. Required module not found

tmp/assets/javascripts/components/Material/Search/Result.jsx:29
 29: import * as lunrRu from 'lunr-languages/lunr.ru';
                             ^^^^^^^^^^^^^^^^^^^^^^^^ lunr-languages/lunr.ru. Required module not found
```

**2. Search highlighting not working for Russian**

**Comment by squidfunk** [comment]
I think the best way would be to integrate it with the general I18N capabilities of Material. When the language is set and differs from English, the theme will autoload the additional stemmers/languages., so no configuration is necessary. I have this on my roadmap, but cannot say when it is finished.

**Comment by monarci** [comment]
Thanks for the help and information, I'll try it out locally and see how it goes.

**Comment by squidfunk** [comment]
Implemented in #347, released with 1.7.0.
